app/models/parser/twitter_profile.rb

```diff
@@ -9,6 +9,8 @@ def type
 
       def patterns
         [
+          /^https?:\/\/(www\.)?twitter\.com\/(?<username>[\w\d]+)(\?+.*)$/,
+          /^https?:\/\/(0|m|mobile)\.twitter\.com\/(?<username>[\w\d]+)(\?+.*)$/,
           /^https?:\/\/(www\.)?twitter\.com\/(?<username>[^\/]+)$/,
           /^https?:\/\/(0|m|mobile)\.twitter\.com\/(?<username>[^\/]+)$/
```

Comment:
Question: These two regular expressions are still too broad and will keep matching things we don't want, right? But it can be a separate ticket, where we follow this: https://help.twitter.com/en/managing-your-account/twitter-username-rules#error
```
Your username cannot be longer than 15 characters. Your name can be longer (50 characters) or shorter than 4 characters, but usernames are kept shorter for the sake of ease.

A username can only contain alphanumeric characters (letters A-Z, numbers 0-9) with the exception of underscores, as noted above. Check to make sure your desired username doesn't contain any symbols, dashes, or spaces.
```

Reply:
Let's leave it as is for now, and refine as we notice those cases (like we're handling in this ticket).